bedrock/base/management/commands/update_www_config.py

```diff
@@ -69,3 +88,44 @@ def handle(self, *args, **options):
 
         self.output("Saved latest git repo state to database")
         self.output("Done!")
+
+    def set_monitor_waitlist(self):
+        # Grab the existing monitor waitlist value so if something fails with the API, we don't lose it.
+        try:
+            obj = ConfigValue.objects.get(name=settings.MONITOR_SWITCH_WAITLIST)
+        except ConfigValue.DoesNotExist:
+            obj = None
+
+        if not settings.MONITOR_ENDPOINT or not settings.MONITOR_TOKEN:
+            # Nothing to do, leave everything as-is.
+            return obj
+
+        resp = requests.get(
+            settings.MONITOR_ENDPOINT, headers={"Content-Type": "application/json", "Authorization": f"Bearer {settings.MONITOR_TOKEN}"}
+        )
+        if resp.status_code != 200:
+            self.output(f"Error getting monitor data: {repr(resp)}")
+            return obj
+
+        monitor_waitlist_value = settings.MONITOR_SWITCH_WAITLIST_DEFAULT
+        data = resp.json()
+        if data.get("success") is True and (message := data.get("message")):
```

Comment:
style nit: don't often see `is True`, usually just `if data.get("success")` is sufficient.

Reply:
Yeah, I just wanted to protect against it returning something like `{"success": "false"}` and that would be truth-y.

Reply:
good point 👍